lib/ff_config.h

```diff
@@ -52,6 +53,9 @@ struct ff_port_cfg {
     char *broadcast;
     char *gateway;
     char *pcap;
+
+    int nb_lcores;
+    uint16_t lcore_list[DPDK_MAX_LCORE];
```

Comment:
use `RTE_MAX_LCORE` instead.

Reply:
if use `RTE_MAX_LCORE`, the `rte_config.h` must be included to `ff_config.h` and it will cause compile error.

Reply:
well, some freebsd source files include the `ff_config.h`, cause the compiling issuse, so keep it this way.
And i'll look at the other codes tomorrow.

Reply:
Actually we can make `lcore_list` as a pointer, and allocate memory when initialize `struct ff_port_cfg`, but I don't like this approach.

Reply:
yeap.